web/templates/guest_visible_users_message.hbs

```diff
@@ -0,0 +1,6 @@
+<p id="guest_visible_users_message">
+    {{t 'Guests will be able to see {user_count} users in their channels when they join.'}}
+    <a id="guest_help_link" class="help_link_widget" href="/help/guest-users#configure-whether-guests-can-see-all-other-users" target="_blank" rel="noopener noreferrer">
+        <i class="fa fa-question-circle-o" aria-hidden="true"></i>
+    </a>
+</p>
```

Comment:
Why isn't this just including the `web/templates/help_link_widget.hbs` template?